styles/normalize.css

```diff
@@ -0,0 +1,461 @@
+/*! normalize.css v5.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Is that better? Should I do it that way or is that just a different way of doing it?